config/initializers/high_voltage.rb

```diff
@@ -0,0 +1,3 @@
+HighVoltage.configure do |config|
+	config.home_page = 'show'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.